cs3api4lab/utils/file_utils.py

```diff
@@ -46,10 +46,58 @@ def normalize_path(path):
             path = '/'
         return path
 
+    @staticmethod
+    def get_my_shares_path_prefix(config):
+        if config.mount_dir == "/":
+            return config.home_dir + "/" + config.shared_folder
+        else:
+            return "/" + config.shared_folder
+
+    @staticmethod
+    def get_share_path_prefix(config):
```

Comment:
@gmgigi96 can you check this and the rest? Is it ok to assume shares will be in `/reva/etc`?
Also, how is this handle drives in revacmd? Because it only returns paths without any drives, AFAIR, so do we also do these replaces there?

Reply:
On cernbox the shares path are "/eos/user/x/...", on local reva setup the shares paths are "/reva/user/...", I am unable to make it more generic